### ISSUE TITLE
Conditional Compilation

### ISSUE THREAD

**Issue by wqweto** [WaynePhillipsEA/twinbasic#229]
**Describe the bug**
Project level introduced Conditional Compilation consts are not resolved on preprocessor `#Const` lines

**To Reproduce**
1. In Project Settings use in Conditional Compilation this
```
MY_TEST=1
```
2. This test code fails to run
```
Module TestModule
    #Const ImplTest = (MY_TEST=1)

    #If ImplTest Then
    Public Sub Main()
    End Sub
    #End If
End Module
```

**Additional context**
The project settings page says "Enter a comma delimited list of name=value pairs used by the compiler during conditional compilation evaluation". Does this deliberately differ from VBA/VB6 colon (`:`) separated list?

Btw, the consts from project settings are always Variants of String sub-type while `#Const` introduced can be Variants of any sub-ype (e.g. `#Const ImplTest = (MY_TEST=1)` must be a Boolean sub-type (the result of comparing a string to an integer literal).

**Comment by WaynePhillipsEA** [comment]
Project level compilation conditional arguments are not yet hooked up, so they are being completely ignored.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.2532, thanks for reporting.  Any errors in the project-level compilation arguments are not gracefully handled yet, but they are parsed and passed to the compiler correctly now.
